app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    store = {}
+
+    def inner(*args) -> (int, float):
```

Comment:
This function can return any value
```suggestion
    def inner(*args) -> Any:
```